pkg/agent/containerd/runtimes_test.go

```diff
@@ -205,7 +205,7 @@ func Test_UnitFindContainerRuntimes(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			foundRuntimes := runtimeConfigs{}
-			findContainerRuntimes(tt.args.root, tt.args.potentialRuntimes, tt.args.locationsToCheck, foundRuntimes)
+			findWasiRuntimes(tt.args.root, foundRuntimes)
```

Comment:
This isn't the right function to test in this file.
```suggestion
			grepRuntimes(tt.args.root, foundRuntimes)
```